frontend/src/components/common/ExpandableTextList.test.tsx

```diff
@@ -0,0 +1,151 @@
+import axios from 'axios';
+import MockAdapter from 'axios-mock-adapter';
+import { createMemoryHistory } from 'history';
+import { IProperty } from 'interfaces';
+import noop from 'lodash/noop';
+import { mockProperties } from 'mocks/filterDataMock';
```

Comment:
as discussed previously we should turn mock objects into functions to avoid test modifying the source object

Reply:
not on this PR though